### ISSUE TITLE
Datasets

### ISSUE THREAD

**Issue by Alice-302** [scuwyh2000/RandBox#14]
Are the data sets used in the paper the same as those used in ORE (Towards Open World Object Detection)? Can I directly use the link to the dataset provided in ORE?

**Comment by scuwyh2000** [comment]
> Are the data sets used in the paper the same as those used in ORE (Towards Open World Object Detection)? Can I directly use the link to the dataset provided in ORE?

Yes, the same. I think directly use the link from ORE works.

**Comment by Alice-302** [comment]
The results obtained during training (run.sh) are basically the same as those obtained from the validation set in run_eval.sh. is this normal? Is the validation set the same as the test set?

**Comment by scuwyh2000** [comment]
> The results obtained during training (run.sh) are basically the same as those obtained from the validation set in run_eval.sh. is this normal? Is the validation set the same as the test set?

Hi, I followed the setting from previous works (ORE and OWDETR). I think for this benchmark, it's the same.
